src/app/lib/config/services/igbo.js

```diff
@@ -26,6 +26,8 @@ const igbo = {
   twitterSite: '@BBCNews', // to be updated
   translations: {
     home: 'Akụkọ',
+    currentPage: 'Current page',
```

Comment:
Is there an issue to get these updated with translations?

Reply:
There isn't, product was waiting to know if we need more translations apart from these.

Reply:
Can you write the issue anyway plz, whenever we get the translations we can use the issue :)